modules/ROOT/pages/dw-core-functions-filterobject.adoc

```diff
@@ -95,9 +95,38 @@ output application/json
 { "1": "a" }
 ----
 
+=== Example
```

Comment:
TODO (DEV): OK with adding this example? It was in v4.3 docs but not DWL.

TODO (DUKE): add to Core.dwl if approved.